### ISSUE TITLE
Using tfjs-node-gpu on a Cuda compute capability 3.0 GPU

### ISSUE THREAD

**Issue by AdamHordines** [tensorflow/tfjs#3574]
#### TensorFlow.js version
@tensorflow/tf-js-node-gpu: 2.0.1

#### Browser version
Node v13.10.1
Windows 10

#### Describe the problem or feature request
The script logs `tensorflow/core/common_runtime/gpu/gpu_device.cc:1700] Ignoring visible gpu device (device: 0, name: GeForce GTX 760, pci bus id: 0000:1b:00.0, compute capability: 3.0) with Cuda compute capability 3.0. The minimum required Cuda capability is 3.5.` Is it possible to use tf-js-node-gpu on a card like the GeForce GTX 760 that only has Cuda compute capability 3.0 and if so how?

I noticed that there were some instructions for running Tensorflow on a GPU with CUDA Compute Capability 3.0 by building from source [here](https://www.tensorflow.org/install/gpu#linux_setup), but this is for the Python version of Tensorflow I believe. It also specifically mentions that method is for linux (even though there are also instructions for building from source on other platforms) and I am running on Windows 10.

#### Code to reproduce the bug / link to feature request
Sample code from [the setup page](https://www.tensorflow.org/js/tutorials/setup#nodejs_setup)
```javascript
const tf = require('@tensorflow/tfjs');

// Optional Load the binding:
// Use '@tensorflow/tfjs-node-gpu' if running with GPU.
require('@tensorflow/tfjs-node-gpu');

// Train a simple model:
const model = tf.sequential();
model.add(tf.layers.dense({units: 100, activation: 'relu', inputShape: [10]}));
model.add(tf.layers.dense({units: 1, activation: 'linear'}));
model.compile({optimizer: 'sgd', loss: 'meanSquaredError'});

const xs = tf.randomNormal([100, 10]);
const ys = tf.randomNormal([100, 1]);

model.fit(xs, ys, {
  epochs: 100,
  callbacks: {
    onEpochEnd: (epoch, log) => console.log(`Epoch ${epoch}: loss = ${log.loss}`)
  }
});
```

**Comment by rthadur** [comment]
@AdamHordines did you get chance to go over this instructions [here](https://medium.com/@naarkie/using-tensorflow-gpu-on-a-compute-3-0-graphics-card-in-windows-4184f4228fe9) ?

**Comment by AdamHordines** [comment]
I did not follow this guide, but I did install the necessary software requirements [listed here](https://www.tensorflow.org/install/gpu#software_requirements). The Medium article seems to be installing Tensorflow 1.8 for Python. While I can work with this (thanks for directing me to it), I wonder if it is possible to get a Node version working (the Python installation doesn't affect the Node package, does it?), and if this is the latest version of Tensorflow that is compatible with Cuda compute capability 3.0 GPUs.

I'll post an update if I can successfully follow the instructions you linked.
